packages/fast-colors/src/color-converters.ts

```diff
@@ -16,7 +16,12 @@ import { degreesToRadians, radiansToDegrees } from "./math-utilities";
 // https://web.stanford.edu/~sujason/ColorBalancing/adaptation.html
 // http://brucelindbloom.com/index.html
 
-// The alpha channel of the input is ignored
+/**
+ * Get the relative luminance of a color.
+ * Adjusts the color to sRGB space, which is necessary for the WCAG contrast spec.
+ * The alpha channel of the input is ignored.
+ * @param rgb The input color
+ */
 export function rgbToLuminance(rgb: ColorRGBA64): number {

```

Comment:
Do you think we should deprecate this name and rename to `rgbToRelativeLuminance`?